src/include/session.h

```diff
@@ -66,6 +66,10 @@ struct __wt_session_impl {
 
 	WT_DATA_HANDLE *dhandle;	/* Current data handle */
 
+	WT_DATA_HANDLE *sync_dhandle;	/* Data handle being synced */
+#define	WT_SESSION_TREE_SYNC(session)					\
+	(session->dhandle == session->sync_dhandle)
+
```

Comment:
This macro could use a better name but I haven't thought of one yet.  Suggestions welcome.

Reply:
I'm with @agorrod, I don't have a better name.